observability/readme.md

```diff
@@ -27,6 +28,20 @@ Building observable systems enables development teams at CSE to measure how well
 
 Read on how Observability can be implemented in [Machine Learning](observability/ml-observability.md) engagements effectively during Model tuning, experimentation and deployment.
 
+## Observability As Code
+
+As much as possible, configuration and management of observability assets such as cloud resource provisioning, monitoring alerts and dashboards must be managed as code. Infrastructure as Code is acheived using any one of Terraform / Ansible / ARM Templates
+
+### Why
+
+* Repeatable and automatable, avoids manual configuration of monitoring alerts and dashboards from scratch across environments.
+
+* Configured dashboards help troubleshoot errors during integration and deployment (CI/CD)
```

Comment:
Why is this better with Observability as code than manually configured observability?